spec/features/users/02_user_signs_out_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+feature "user signs out", %{
+  As an authenticated user
+  I want to sign out
+  So that I can leave the app.
+} do
+  # Acceptance Criteria
+  # [ ] Link to sign out exists in the nav-bar when logged in
+  # [ ] Clicking Sign Out show message you have signed out
+  # [ ] Link to Sign In appears after signing out
+  scenario "user sees sign in and sign up links" do
```

Comment:
This spec says its for a user sign out feature, but this is testing signing in and signing up. This should be in its own test, created on its own branch, pushed in its own PR. I'll let it slide being on this branch/ PR this time but won't in the future. It does need to be refactored into its own test however, or rename this file to more accurately describe whats being tested in it